<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfacciaRegistrazione</class>
 <widget class="QWidget" name="InterfacciaRegistrazione">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>725</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:rgb(113, 113, 113);
}

QPushButton
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0.507177 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 127, 255));
	font: bold 16pt &quot;Kinnari&quot;;
	color: rgb(255, 170, 0);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;
}

QPushButton:hover
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.507177 rgba(255, 170, 0, 255)); 
	color: rgb(0, 0, 0);
}


QPushButton:hover:!pressed
{
	background-color:qlineargradient(spread:pad, x1:0.244, y1:1, x2:0.837, y2:0.227273, stop:0 rgba(255, 255, 127, 255), stop:0.507177 rgba(0, 0, 0, 255));
	color: rgb(255, 170, 0);
}

#RegistrazioneLabel
{
	font: bold italic 32pt &quot;Kinnari&quot;;
	color: rgb(255, 85, 0);
	background-color:rgb(113, 113, 113);
}

#DatiPersonaliLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#ContattiLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#CambiaPasswordLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#DatiLoginLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#ScegliAccountLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#LingueLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#EsperienzeLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

QLabel
{
	font: bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color:rgb(57, 57, 57);
}

QLineEdit
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 170, 0);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

QTextEdit
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 170, 0);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

QRadioButton
{
	font: bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color:rgb(57, 57, 57);
}

QDateEdit
{ 
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 170, 0);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

QGroupBox
{
	background-color:rgb(57, 57, 57);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;
}

#line2
{
	background-color:rgb(57, 57, 57);
}</string>
  </property>
  <widget class="QLabel" name="RegistrazioneLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>-10</y>
     <width>270</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrazione</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>730</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="DatiPersonaliLabel">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>60</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Dati Personali</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>690</width>
     <height>195</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>2</y>
      <width>16</width>
      <height>190</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="NomeLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>116</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NomeLineEdit">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>11</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>11</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="CognomeLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>47</y>
      <width>116</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cognome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CognomeLineEdit">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>47</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>47</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="DataNascitaLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>83</y>
      <width>116</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Nascita</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="DataNascitaDateEdit">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>83</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>83</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="LuogoNascitaLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>119</y>
      <width>116</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Luogo Nascita</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LuogoNascitaLineEdit">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>119</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>119</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="TelefonoLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>155</y>
      <width>116</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefono</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TelefonoLineEdit">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>155</y>
      <width>180</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>155</y>
      <width>16</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmailLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>11</y>
      <width>134</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailLineEdit">
    <property name="geometry">
     <rect>
      <x>484</x>
      <y>11</y>
      <width>179</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>11</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="IndirizzoLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>47</y>
      <width>134</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Indirizzo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IndirizzoLineEdit">
    <property name="geometry">
     <rect>
      <x>484</x>
      <y>47</y>
      <width>179</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>47</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="CittaLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>83</y>
      <width>134</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Città</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CittaLineEdit">
    <property name="geometry">
     <rect>
      <x>484</x>
      <y>83</y>
      <width>179</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>83</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="ProvinciaLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>119</y>
      <width>134</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Provincia</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProvinciaLineEdit">
    <property name="geometry">
     <rect>
      <x>484</x>
      <y>119</y>
      <width>179</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>119</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="TitoloStudioLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>155</y>
      <width>134</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Titolo Di Studio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TitoloStudioLineEdit">
    <property name="geometry">
     <rect>
      <x>484</x>
      <y>155</y>
      <width>179</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>155</y>
      <width>16</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="DatiLoginLabel">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>285</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Dati Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>315</y>
     <width>335</width>
     <height>122</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="UsernameLabel">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="UsernameLineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="PasswordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="PasswordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="ConfermaPasswordLabel">
      <property name="text">
       <string>Conferma Password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="ConfermaPasswordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ScegliAccountLabel">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>285</y>
     <width>305</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Scegli Il Tipo Di Account *</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>315</y>
     <width>135</width>
     <height>122</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="BasicRadioButton">
      <property name="text">
       <string>Basic</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="BusinessRadioButton">
      <property name="text">
       <string>Business</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="ExecutiveRadioButton">
      <property name="text">
       <string>Executive</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="LingueLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>437</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Lingue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="LingueTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>467</y>
     <width>265</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="EsperienzeLabel">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>437</y>
     <width>225</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Esperienze Lavorative</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="EsperienzeLavorativeTextEdit">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>467</y>
     <width>265</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RegistratiButton">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>675</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Registrati</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnnullaButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>675</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>285</y>
     <width>10</width>
     <height>387</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
